config/locales/en.yml

```diff
@@ -698,6 +698,16 @@ en:
       delete_confirm: "Are you sure you want to delete the reason: %{name_sv} (%{name_en})?"
 
 
+  shf_mailer:
+    greeting: "Hello %{greeting_name}"
+    shf_signoff: "Woof!"
+    shf_signature: "Sveriges Hundföretagare (SHF) [Swedish Dog Industry Association]"
+    shf_signature_url: "http://sverigeshundforetagare.se"
+
+    footer:
+      text: "Please don't reply to this email. (Should reply email be set to info@... or member-help@... ?)  Received this by mistake ....? <what should go here?>"
```

Comment:
I think we probably should set the reply-to for the reset-password email to something like one of your suggestions (`info@ ....`, for instance).

Then, in this footer, we could say "Please don't reply to this email. If you have any questions, please send those to `info@ ....`."    (the email address would be a `mail_to` link, of course.)